kbase-extension/static/kbase/js/common/jobCommChannel.js

```diff
@@ -268,53 +277,43 @@ define([
-                case 'run_status':
-                    // Send job status notifications on the default channel,
-                    // with a key on the message type and the job id, sending
-                    // a copy of the original message.
-                    // This allows widgets which are interested in the job
-                    // to subscribe to just that job, and nothing else.
-                    // If there is a need for a generic broadcast message, we
-                    // can either send a second message or implement key
-                    // filtering.
-                    this.sendBusMessage(CELL, msgData.cell_id, 'run-status', msgData);
+                case BACKEND_RESPONSES.RUN_STATUS:
+                    this.sendBusMessage(CELL, msgData.cell_id, RESPONSES.RUN_STATUS, msgData);
                     break;
 
-                case 'result':
-                    this.sendBusMessage(CELL, msgData.address.cell_id, 'result', msgData);
+                case BACKEND_RESPONSES.RESULT:
+                    this.sendBusMessage(CELL, msgData.address.cell_id, RESPONSES.RESULT, msgData);
                     break;
 
                 // JOB messages
+                // Send job-related notifications on the default channel,
+                // with a key on the message type and the job id.
+                // This allows widgets which are interested in the job
+                // to subscribe to just that job, and nothing else.
+                // If there is a need for a generic broadcast message, we
+                // can either send a second message or implement key
+                // filtering.
+
+                // errors
                 case 'job_comm_error':
                     console.error('Error from job comm:', msg);
                     if (!msgData) {
                         break;
                     }
                     jobId = msgData.job_id;
                     switch (msgData.source) {
-                        case 'cancel_job':
-                            this.sendBusMessage(JOB, jobId, 'job-cancel-error', {
-                                jobId,
-                                message: msgData.message,
-                            });
-                            this.sendBusMessage(JOB, jobId, 'job-canceled', {
+                        case JOB_REQUESTS.LOGS:
+                            this.sendBusMessage(JOB, jobId, RESPONSES.LOGS, {
                                 jobId,
                                 error: msgData.message,
                             });
-                            break;
-                        case 'job_logs':
-                        case 'job_logs_latest':
                             this.sendBusMessage(JOB, jobId, 'job-log-deleted', {
                                 jobId,
                                 message: msgData.message,
                             });
-                            this.sendBusMessage(JOB, jobId, 'job-logs', {
```

Comment:
this hasn't been deleted, I just switched the order of this message and the prior one.